test-harness/src/test/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfiguratorTest.java

```diff
@@ -96,7 +96,7 @@ public void exportYaml() throws Exception {
         assertEquals(Util.toYamlString(map.get("bar")).trim(), "true");
         assertEquals(Util.toYamlString(map.get("qix")).trim(), "42");
         assertEquals(Util.toYamlString(map.get("zot")).trim(), "\"zot\"");
-        assertEquals(Util.toYamlString(map.get("dbl")).trim(), "12.34");
+        assertEquals(Util.toYamlString(map.get("dbl")).trim(), "\"12.34\"");
```

Comment:
could we add a test that this is databound properly for a double value? e.g. the test from https://github.com/jenkinsci/configuration-as-code-plugin/pull/1560/files

Reply:
Sure! Good idea. I'll merge it in to this PR.